model_hub/model_hub/mmdetection/_callbacks.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, context: det_torch.PyTorchTrialContext):
 
     @property
     def optimizer(self) -> Dict[int, Any]:
-        return {i: opt for i, opt in enumerate(self.context.optimizers)}
+        return dict(enumerate(self.context.optimizers))
```

Comment:
is this change intentional?

Reply:
it is but I think this was resolved in master. it shouldn't show up after I updated the branch. this is gone now